config/krakend/settings/workflow_proxy_settings.json

```diff
@@ -46,14 +46,14 @@
             "url_pattern": "/metadata/workflow/{name}"
         },
         {
-            "endpoint": "/api/uniflow/metadata/workflow/{a}/{b}",
+            "endpoint": "/api/uniflow/metadata/workflow/{name}/{version}",
             "method": "DELETE",
-            "url_pattern": "/metadata/workflow/{a}/{b}"
+            "url_pattern": "/metadata/workflow/{name}/{version}"
         },
         {
-            "endpoint": "/api/uniflow/workflow/{a}",
```

Comment:
this endpoint also cover /api/uniflow/workflow/:workflowId?includeTask={true|false}, which is included in public API proposal. is that ok?